math/minuit2/inc/Minuit2/MinimumError.h

```diff
@@ -68,24 +64,23 @@ class MinimumError {
    }
 
    double Dcovar() const { return fPtr->fDCovar; }
-   bool IsAccurate() const { return Dcovar() < 0.1; }
-   bool IsValid() const { return fPtr->fValid; }
-   bool IsPosDef() const { return fPtr->fPosDef; }
-   bool IsMadePosDef() const { return fPtr->fMadePosDef; }
-   bool HesseFailed() const { return fPtr->fHesseFailed; }
-   bool InvertFailed() const { return fPtr->fInvertFailed; }
-   bool IsAvailable() const { return fPtr->fAvailable; }
+   Status GetStatus() const { return fPtr->fStatus; }
+
+   bool IsValid() const { return IsAvailable() && (IsPosDef() || IsMadePosDef()); }
+   bool IsAccurate() const { return IsPosDef() && Dcovar() < 0.1; }
```

Comment:
Before, this just checked Dcovar() < 0.1. Checking IsPosDef() is not necessary they way the ctors are set up, but this seems more clear and safer.